test/common/stats/stat_test_utility.h

```diff
@@ -72,6 +77,79 @@ class MemoryTest {
   const size_t memory_at_construction_;
 };
 
+// Helper class to use in lieu of an actual Stats::Store for doing lookups by
+// name. The intent is to remove the deprecated Scope::counter(const
+// std::string&) methods, and always use this class for accessing stats by
+// name.
+//
+// This string-based lookup wrapper is needed because the underlying name
+// representation, StatName, has multiple ways to represent the same string,
+// depending on which name segments are symbolic (known at compile time), and
+// which are dynamic (e.g. based on the request, e.g. request-headers, ssl
+// cipher, grpc method, etc). We need to do name-matching based only on strings
+// to avoid having the tests know which segments are dynamic.
+//
+// Note that this class does not implement the of Stats::Store interface, but
```

Comment:
typo "the of"

Reply:
eliminated that paragraph as now the class does implement Stats::Store.